generators/server/templates/src/main/java/package/Application.java.ejs

```diff
@@ -58,7 +59,7 @@ import java.util.Collection;
     excludeFilters = @ComponentScan.Filter(type = FilterType.ASSIGNABLE_TYPE, classes = OAuth2InterceptedFeignConfiguration.class)
 )
 <%_ } _%>
-@SpringBootApplication
+@SpringBootApplication(exclude = JmxAutoConfiguration.class)
```

Comment:
Are you sure this needs to be excluded ? some people might want to activate it later.

Reply:
Yes I'll revert this: I'm just following the blog post here, I'm not finding any great improvements by removing JMX. Then, I see it also as a security risk, so let's remove it be default.

(I can't write here what I've done with JMX in the past, but trust me there were several very high security issues because JMX was left opened)